### PR TITLE
User-create-entry - Heerak Moon

### PR DESCRIPTION
**Pull request by hmoon2**
https://user-images.githubusercontent.com/113075380/210004430-c8124e41-ef62-4cab-92a6-d2e12d7e2d11.mp4

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -17,11 +17,48 @@
 </head>
 <body>
   <header>
-    <div class="container">
+    <div class="container margin-none">
+      <div class="row">
+        <div class="column-full purple-background">
```
Comment (codingcodymiller) [inline index.html:22]: You should put the `purple-background` class on the `header` so it fills the whole viewport

File: index.html [hunk 2]
```diff
@@ -17,11 +17,48 @@
 </head>
 <body>
   <header>
-    <div class="container">
+    <div class="container margin-none">
```
Comment (codingcodymiller) [inline index.html:20]: Please don't remove margin from the `container` class

File: index.html [hunk 3]
```diff
@@ -17,11 +17,48 @@
...
   </header>
   <main class="container">
-    <div data-view="entry-form"></div>
+    <div data-view="entry-form">
+      <form>
+        <div class="row">
+          <h1 class="padding-10 column-full">New Entry</h1>
+        </div>
+        <div class="row space-between">
+          <div class="column-half no-padding">
```
Comment (codingcodymiller) [inline index.html:35]: Please don't remove padding from the columns.

File: index.html [hunk 4]
```diff
@@ -17,11 +17,48 @@
...
+            <img id="image" class="padding-10 img-fill" src="images/placeholder-image-square.jpg" alt="no image, placeholder image">
+          </div>
+          <div class="column-half no-padding">
```
Comment (codingcodymiller) [inline index.html:38]: Please don't remove padding from the columns.

File: index.html [hunk 5]
```diff
@@ -17,11 +17,48 @@
...
+            <div class="padding-5">
+              <h4 class="padding-5">Title</h4>
```
Comment (codingcodymiller) [inline index.html:40]: this should be a `label`

File: index.html [hunk 6]
```diff
@@ -17,11 +17,48 @@
...
+              <input id="title" class="input-padding" type="text" required="required">
+            </div>
+            <div class="padding-10">
+              <h4 class="padding-5">Photo URL</h4>
```
Comment (codingcodymiller) [inline index.html:44]: this should be a `label`

File: index.html [hunk 7]
```diff
@@ -17,11 +17,48 @@
...
+        </div>
+        <div class="row">
+          <div class="padding-10 column-full">
+            <h4 class="padding-5">Notes</h4>
+            <div class="padding-5">
+              <textarea class="input-padding" name="notes" id="notes" cols="30" rows="10" required="required"></textarea>
+            </div>
+          </div>
+        </div>
+        <div class="row padding-10 column-full button-right">
```
Comment (codingcodymiller) [inline index.html:57]: No one element should be both a `row` and a `column`. Give this `row` a child with the `column-full` class.

File: css/styles.css [hunk 8]
```diff
@@ -0,0 +1,102 @@
...
+}
+
+.margin-none {
+  margin: 0;
+  max-width: 100%;
+}
+
+.no-padding {
+  padding: 0;
+  width: 440px;
```
Comment (codingcodymiller) [inline css/styles.css:78]: There should be no `px` width set on anything except the `container` class.

File: css/styles.css [hunk 9]
```diff
@@ -0,0 +1,102 @@
...
+  justify-content: space-between;
+}
+
+@media screen and (min-width: 768px) {
+  * {
+    margin: 0;
+  }
+
+  form {
+    width: 900px;
```
Comment (codingcodymiller) [inline css/styles.css:91]: There should be no `px` width set on anything except the `container` class.

File: css/styles.css [hunk 10]
```diff
@@ -0,0 +1,102 @@
...
+  }
+
+  img {
+    max-height: 400px;
+  }
+
+  .width {
+    width: 900px;
+    margin: auto;
+  }
```
Comment (codingcodymiller) [inline css/styles.css:101]: There should be no `px` width set on anything except the `container` class.

File: css/styles.css [hunk 11]
```diff
@@ -0,0 +1,102 @@
...
+}
+
+.white-text {
+  font-weight: 300;
+  color: white;
+}
+
+.width {
+  width: 350px;
+}
```
Comment (codingcodymiller) [inline css/styles.css:49]: There should be no `px` width set on anything except the `container` class.

File: css/styles.css [hunk 12]
```diff
@@ -0,0 +1,102 @@
...
+  border-radius: 5px;
+  padding: 5px 2px;
+  border: none;
+  margin: 0;
+  background-color: rgb(82, 23, 109);
+}
+
+form {
+  width: 350px;
+}
```
Comment (codingcodymiller) [inline css/styles.css:23]: There should be no `px` width set on anything except the `container` class.

File: css/styles.css [hunk 13]
```diff
@@ -0,0 +1,102 @@
+* {
+  font-family: sans-serif;
+  width: 100%;
+  margin: auto;
+  padding: 0;
```
Comment (codingcodymiller) [inline css/styles.css:5]: You shouldn't do this, this is a _**really**_ major change

File: css/layout.css [hunk 14]
```diff
@@ -5,7 +5,7 @@
 .container {
   max-width: 575px;
   margin: 0 auto;
-  padding: 0 1.5rem;
+  padding: 0;
```
Comment (codingcodymiller) [inline css/layout.css:8]: Please leave this as `padding: 0 1.5rem;`

**Review by codingcodymiller — CHANGES_REQUESTED** [review]

**Comment by hmoon2** [comment]
https://user-images.githubusercontent.com/113075380/210022606-b6c85d51-77ef-44e5-98df-cb796aba83cb.mp4

**Review by codingcodymiller — APPROVED** [review]

**Pull request merged**